src/pages/Leaderboard/Leaderboard.tsx

```diff
@@ -13,15 +15,32 @@ import { Wrapper } from 'uicomponents/Wrapper/styled';
 
 import * as Styled from './styled';
 
-export const Leaderboard: FC = () => (
-    <Wrapper>
-        <Title>LeaderBoard</Title>
-        <Styled.LeaderTable>
-            <TableHead />
-            <TableBody elements={fakeElements} />
-        </Styled.LeaderTable>
-        <BaseButton size="s" view="primaryFlat">
-            <Link to="/">Домой</Link>
-        </BaseButton>
-    </Wrapper>
-);
+export const Leaderboard: FC = () => {
+    const [elements, setElements] = React.useState(null);
+    React.useEffect(() => {
```

Comment:
Это шо такое?)
Давай заимпортим отдельно
```suggestion
    useEffect(() => {
```